README.md

```diff
@@ -1,6 +1,8 @@
 # Data Visualization in Python
 
-![285d68dd-charts-1024x581](https://github.com/Debaditya-Som/Data-Visualization-in-Python/assets/121785700/d40a65d0-ec85-462a-9475-4982a375d067)
+<img src="https://github.com/Debaditya-Som/Data-Visualization-in-Python/assets/121785700/d40a65d0-ec85-462a-9475-4982a375d067" alt="drawing"  width="510"/>
+
+
```

Comment:
It appears that the changes proposed in this PR already exist in the repository.  If there are specific reasons for resubmitting these changes, please provide additional context or explain any differences from the existing code.